packages/experimental/Button/src/Button.types.ts

```diff
@@ -105,6 +105,7 @@ export interface ButtonProps extends ButtonCoreProps {
    * - 'subtle': Minimizes emphasis to blend into the background until hovered or focused.
```

Comment:
Worth noting that we do still have content?: string in the props still even though you appear to have removed support for it.

Reply:
I removed it for the main Button. But I haven't done it yet for Compound, Toggle button and FAB. If this solution is good for us, then I'll remove it everywhere